src/content/getting-started/developer-tools/cli.md

```diff
@@ -87,34 +42,37 @@ If you installed the Particle CLI through the installer, it will periodically up
 To force it to update, run the installer script again or enter this command:
 
 ```sh
-# how to update the installed CLI
 $ particle update-cli
 ```
 
-If the CLI is outputing unexpected errors after an update, delete the `~/.particle` (Mac OS and Linux) or `C:\Users\<username>\AppData\Local\particle` directory and run the installer script again to start over.
+#### Disabling automatic updates
 
-**To prevent the Particle CLI from automatically updating, set the environment variable `PARTICLE_DISABLE_UPDATE=true` for your system. Use `particle update-cli` to manually update.**
+If you wish to prevent automatic updates from occurring you can do so with the following command. This might be appropriate on an assembly line where you want to keep the configuration and binaries constant, however in general we recommend always using the current version.
 
-If you installed manually using `npm install`, you can upgrade by running the same command you used to install the tool.
+```sh
+$ particle update-cli --disable-updates
+```
+
+#### Reenabling automatic updates
 
-### Running from source (advanced)
+To resume updates and update if necessary now:
+
+```sh
+$ particle update-cli --enable-updates
+```
 
-To grab the CLI source and play with it locally
+#### Installing a specific version
```

Comment:
Can you link to the "Using a specific Particle CLI version" page from here as well?

Reply:
Done